### ISSUE TITLE
node error throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map')

### ISSUE THREAD

**Issue by inicefish** [iron-fish/ironfish#1500]
### What happened?

C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\account\accounts.js:318
                            throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map');
                                  ^

Error: nullifierToNote mappings must have a corresponding noteToNullifier map
    at C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\account\accounts.js:318:35
    at async LevelupDatabase.withTransaction (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\storage\database\database.js:39:28)

### Graffiti

inicefish

**Comment by centrd** [comment]
I'm having the same issue on one of my nodes. 

> Version       0.1.35 @ 4bc3765
> Node Name     T777
> Graffiti      T777
> Peer Identity 7762QbY5gXL+daoMOB0cXp6wQ015ZMJrQk9dNwyPoEs=
> Peer Agent    ironfish/0.1.35/4bc3765
> Peer Port     9033
> Bootstrap     test.bn1.ironfish.network
> 
> WebSocket server started at :::9033
> /usr/src/app/node_modules/@ironfish/sdk/build/src/account/accounts.js:318
>                             throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map');
>                                   ^
> 
> Error: nullifierToNote mappings must have a corresponding noteToNullifier map
>     at /usr/src/app/node_modules/@ironfish/sdk/build/src/account/accounts.js:318:35
>     at async LevelupDatabase.withTransaction (/usr/src/app/node_modules/@ironfish/sdk/build/src/storage/database/database.js:39:28)

Mine runs in docker on Ubuntu 20.04. 
I've tried to remove the docker container and redeploy, but no dice.
This started happening with no apparent trigger. The node was running last night, then when I got back to it, it had crashed or exited, and upon trying to restart it, the above noted error shows every time. 

As this is obviously not an isolated case, I'm hoping it can get some attention.

**Comment by centrd** [comment]
I tried a couple more things:

> chain:repair --force

Result: 
Every time I execute the above, it runs for a few seconds, then errors out with: "FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory"

> accounts:rescan --reset
This appears to have executed without errors.

After running the two tasks above a couple times, my node started to sync again. I still don't know what happened or why, and I'm not confident that everything is fixed. But it's synching now, so I will keep an eye on it and see if it can reach 100% sync.

**Comment by Shekelme** [comment]
Especially often this error began to manifest itself after upgrading to version 0.1.37 during the simple operation of the node in the mode of keeping synchronization up to date.
```
Error: nullifierToNote mappings must have a corresponding noteToNullifier map
    at /usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:355:31
    at async LevelupDatabase.withTransaction (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/storage/database/database.js:39:28)
    at async Accounts.removeTransaction (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:337:9)
    at async Accounts.expireTransactions (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:646:17)
    at async Accounts.eventLoop (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:160:9)
```

I use NPM installation method, Node.js v16.15.1, Ubuntu server 20.04.4 LTS, AMD Ryzen Threadripper PRO 3955WX 16-Cores, 512GB RAM

**Comment by centrd** [comment]
I started getting this error too after upgrading yesterday. It's running on Ubuntu 20.04 and has lots of RAM and CPU resources available.

**Comment by centrd** [comment]
```
docker run --rm --tty --interactive --network host --volume ~/.ironfish:/root/.ironfish ghcr.io/iron-fish/ironfish:latest chain:repair --force
Opening node... done.

⚠️ If you start repairing your database, you MUST finish the
process or your database will be in a corrupt state. Repairing
may take 28m 38s or longer.

Are you SURE? (y)es / (n)o: y
Clearing hash to next hash table... done
Clearing Sequence to hash table... done
Repairing head chain tables: [========================================] 72136/72136 100% 7444/sec | soon

Repairing MerkleTrees
Clearing notes MerkleTree... ⢿

<--- Last few GCs --->

[7:0x560d6a0]   251429 ms: Mark-sweep 4050.3 (4137.1) -> 4039.9 (4139.6) MB, 9900.4 / 0.0 ms  (average mu = 0.171, current mu = 0.022) task scavenge might not suc
ceed
[7:0x560d6a0]   258297 ms: Mark-sweep 4053.4 (4139.8) -> 4043.0 (4142.8) MB, 6686.4 / 0.0 ms  (average mu = 0.105, current mu = 0.027) task scavenge might not suc
ceed
[7:0x560d6a0]   269957 ms: Mark-sweep 4055.8 (4142.8) -> 4045.7 (4145.3) MB, 11500.8 / 0.0 ms  (average mu = 0.050, current mu = 0.014) task scavenge might not su
cceed


<--- JS stacktrace --->

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
 1: 0xb09c10 node::Abort() [node]
 2: 0xa1c193 node::FatalError(char const*, char const*) [node]
 3: 0xcf8dde v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
 4: 0xcf9157 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
 5: 0xeb09f5  [node]
 6: 0xec06bd v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
 7: 0xf23884 v8::internal::ScavengeJob::Task::RunInternal() [node]
 8: 0xdb6f6b non-virtual thunk to v8::internal::CancelableTask::Run() [node]
 9: 0xb77a54 node::PerIsolatePlatformData::RunForegroundTask(std::unique_ptr<v8::Task, std::default_delete<v8::Task> >) [node]
10: 0xb798b9 node::PerIsolatePlatformData::FlushForegroundTasksInternal() [node]
11: 0x1564956  [node]
12: 0x1577084  [node]
13: 0x15652a8 uv_run [node]
14: 0xa43dd5 node::SpinEventLoop(node::Environment*) [node]
15: 0xb4bdb6 node::NodeMainInstance::Run(node::EnvSerializeInfo const*) [node]
16: 0xacd592 node::Start(int, char**) [node]
17: 0x7fe7738ebd0a __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
18: 0xa4067c  [node]
Aborted (core dumped)

```

**Comment by Shekelme** [comment]
0.1.38 did not bring us a solution for this...

**Comment by Shekelme** [comment]
0.1.39 also did not bring us a solution...
```
SEND WebRtc ETTSMWW: NewTransaction (8)
SEND WebRtc kaTuKh0: NewTransaction (8)
Updated Accounts Head: 000000000001f7d2dc0698ed4d45afda1ef71d1cc177a0f8fd628fe44a2b5849
/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:351
                        throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map');
                              ^

Error: nullifierToNote mappings must have a corresponding noteToNullifier map
    at /usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:351:31
    at async LevelupDatabase.withTransaction (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/storage/database/database.js:39:28)
    at async Accounts.removeTransaction (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:333:9)
    at async Accounts.expireTransactions (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:674:17)
    at async Accounts.eventLoop (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:158:9)
```

**Comment by lwisne** [comment]
Can you tell us more about the circumstances under which you're still seeing this? Is it every time you run? Or is it intermittent?

**Comment by Shekelme** [comment]
This manifests itself from time to time during the execution of the `ironfish depositAll` command.
Although it is difficult to say whether these things are connected or not.

**Comment by Shekelme** [comment]
Today all my nodes (about 20 instances) has crashed with this error. Here are some log lines:

```
Accepted tx db9d8036686d1c2af87fab4ee2021246c7e2aa7316485845ab2d1f079b82299e, poolsize 1408
RECV WebRtc QUMxzYX: NewTransaction (8) (repeated 3 times)
SEND WebRtc QUMxzYX: NewTransaction (8)
SEND WebSocket X/CQCa2: NewTransaction (8)
SEND WebRtc aV1olbz: NewTransaction (8)
SEND WebRtc QUMxzYX: NewTransaction (8)
SEND WebSocket X/CQCa2: NewTransaction (8)
SEND WebRtc aV1olbz: NewTransaction (8)
RECV WebRtc QUMxzYX: NewTransaction (8)
RECV WebRtc QUMxzYX: NewTransaction (8)
RECV WebRtc QUMxzYX: NewTransaction (8)
RECV WebRtc QUMxzYX: NewTransaction (8)
Invalid transaction 'be8b1fcba433d30a06706e7047bffbfe5e6b715ce57f7c67c371fa57b2f007e8': expired sequence 114667
Invalid transaction 'b86a37ca29292cb63c0ccd089f95b7f389ce9e9bfb69fd508e4e84d9a44a303d': expired sequence 114825
Invalid transaction '363b6dbc8373924ca4a8f918675364ff67469387bacab4fa515a58656bc1aa75': expired sequence 114785
Invalid transaction '6af5977bedaf1d459246f3d041f2f5d4b415d0c097431b6faac69aa71df46057': expired sequence 114687
Invalid transaction 'b1b68b33f3e312cfaa87b343cab9b04f4478ce893f958e65d724cc0eaaf10653': expired sequence 114715
Invalid transaction '4c50de7cb35f17517fd60bb35d6357824daf091207efa28f7975b45f67bf5f1a': expired sequence 114667
RECV WebRtc aV1olbz: NewTransaction (8)
Invalid transaction '4105fa79d8dec8a96e1657e8965f2c12885181765cb6a18ed9be54bb6b087fbf': expired sequence 114671
Invalid transaction '277ac6d2a720ff4fe780b2cec330bcbd5d4b2604de7f8e16f548feff00675e10': expired sequence 114712
RECV WebRtc aV1olbz: NewTransaction (8)
Invalid transaction 'c519ceda4d7d72eb05f7dc9b3c9ede4f507059aed3d1e3436399e6f638df6c40': expired sequence 114785
Invalid transaction '7779f6abbfb4597045e707ac08e71b904aaec479ec12c4bfb9d0bfa9313b6ebb': expired sequence 114785
Invalid transaction 'cc90127216d7f1a0a49a2a2f227ac615edc7bc4f6424fa8e0681867591a6c42a': expired sequence 114785
Invalid transaction '15fef2e8a2ceb68e21438ca4b9d751123f930bdc31db57c7dcdcbb560f8d399f': expired sequence 114699
SEND WebRtc dB7LSSU: NewTransaction (8) (repeated 7 times)
RECV WebRtc dB7LSSU: NewTransaction (8)
Invalid transaction '0eab8a42cd08a04607ef5f1b1f69f44d574140d191513d5b13fde79f8f9bafbb': expired sequence 114785
Invalid transaction '5f80fd0ca597ef87a0008737267cd990da8f744b226dcb422299ce3c234e8700': expired sequence 114785
Invalid transaction '1b8be9f4b83afe8dd7f693c5b00d18da5d51b0b30410337f41500fdd246ef14b': expired sequence 114462
RECV WebRtc aV1olbz: NewTransaction (8)
RECV WebSocket X/CQCa2: NewTransaction (8)
Invalid transaction 'ca1a5a80be35265333f7354022cf6d9b52c6d51792c7d8a9dd6c3433af97e918': expired sequence 114715
Invalid transaction '518cc19317354ac1d886f3a81fddb077ad3d867e18066a8f9ec586b4c8432eda': expired sequence 114740
Invalid transaction '462dd7c300122396a7d0c1da92daeef79b9f4c1ae679c5011df86abeff6059d0': expired sequence 114666
Invalid transaction '8cfd128059f546d10e040784e9b892a8ed52c6d88ae09ca564023963eea1459e': expired sequence 114785
Invalid transaction 'c55960db4d102940b09f7ca6fe75e8713a2d27972ec1344c1bcc406ad8489818': expired sequence 114785
Invalid transaction '1c4e86e262f2e46cff81fddf0138812a7ef0addb846c737aa9f7739a0e17d6de': expired sequence 114776
Invalid transaction '830e9188ab1d3eb83bea15f6f96d670a797379aa78c15f55cebde294b336f981': expired sequence 114785
Invalid transaction '1d94a5299ce2965b8b3b0d7b891f31c5bd77321ff1cd418a83ebd8f02576263d': expired sequence 114785
Invalid transaction '0a9fbb6e5a4260cc06e84a166b07926d27cf7c192aef7d955c67e3d106a0545e': expired sequence 114785
Invalid transaction 'b80137e76dbcc09410d76e8eee06a0ba62ca0ac184f7629d8c1b6cfd5023daab': expired sequence 114750
Invalid transaction 'b8d6b43079adf3995fe8d97733ecf88991189bbda0b3630a6384194197aa385c': expired sequence 114785
Invalid transaction 'e4dd8c9044cafb11e8d613f122342346e978bc2be63ad366f30a1cdd86de48a6': expired sequence 114785
Invalid transaction '4b4cf63222771180f6d35accf95634eb0f7bb179d89cd52baa2dbdb6430ce183': expired sequence 114785
Invalid transaction 'ed60da68bf0a5c456ade81b6b078eebf784ef116db1e31ab96b8cf776a2080ad': expired sequence 114785
Invalid transaction '1ee5cf905f61857189dde4a4e45373e21ba1faf1fca4cd2e2f06434115dfd4a0': expired sequence 114785
Invalid transaction '673ed91bbce30997249dfa67c7a2b72501892b0904df6ca64ea5422fe175cf74': expired sequence 114785
Invalid transaction 'a22d658bdede1cffd0314e1e0759ad6763996f11c53de3af3428a94491ae8bc1': expired sequence 114785
Invalid transaction 'a389c344fb22b1f590ffe429a00be244058e19d37565865a403a5fbda9feba5e': expired sequence 114785
Invalid transaction '36be6cfc9f76c5764a9331f63084b54d26062f8f0b12895375f736bccbb122ec': expired sequence 114785
SEND WebRtc TbORWYE: NewTransaction (8) (repeated 10 times)
RECV WebRtc TbORWYE: NewTransaction (8)
RECV WebRtc TbORWYE: NewTransaction (8)
Invalid transaction '22d5fee28130aea546a405742c98d24cd44b1ee14f115599119ac32dafa7b416': expired sequence 114785
RECV WebRtc TbORWYE: NewTransaction (8)
Invalid transaction '22d5fee28130aea546a405742c98d24cd44b1ee14f115599119ac32dafa7b416': expired sequence 114785
Invalid transaction 'cea20521106feb12b25caab1c0d3d71ee0ed32e60798efaeddf38d185d9be9b7': expired sequence 114854
RECV WebRtc TbORWYE: NewTransaction (8)
Invalid transaction '82344ae6639508c61d3203f1ec7f099ee404c8312c02894928ff9c61351a0a0d': expired sequence 114854
Invalid transaction '2feb3b0230ae9cd305524a61f641ac8be7c0852124d0a4aaac7a492b28a231f0': expired sequence 114785
RECV WebRtc TbORWYE: NewTransaction (8)
Invalid transaction '44d0b0733e91c6860c802eda294a1cde63699e10a2e5f736bc94d7155d741d88': expired sequence 114785
Invalid transaction 'ad3dc2274a9600901748a74db2fa0d3e6956c8d806ebe1a787013507e8afb292': expired sequence 114785
Invalid transaction 'af43449a95ec78622be3458c549d06a10d1d6c1cf4a2f15381c10ad5e8ecb895': expired sequence 114785
Invalid transaction 'ad7c605dea94d4ec98066e698cbabf7956f69039c65d897d7b9fa3228091df52': expired sequence 114785
Invalid transaction '446079bc38cbace3fa6a4d223c54769a1013290ce9110c7a281cab52537e6dd6': expired sequence 114785
Invalid transaction 'cde200915f68b6c4b4375c2d8b9406f6d9c31113bfbdc0fdfc2df14511a987fe': expired sequence 114837
/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:316
                            throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map');
                                  ^

Error: nullifierToNote mappings must have a corresponding noteToNullifier map
    at /usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:316:35
    at async LevelupDatabase.withTransaction (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/storage/database/database.js:39:28)
```
And from another node:

```
Accepted tx c03b1d1d48ba074871aaba5a98decd9ed56f95e65f8aed4fd193675637e5097d, poolsize 1847
SEND WebSocket VfmuQLo: NewTransaction (8)
SEND WebRtc nEkSSRl: NewTransaction (8)
SEND WebRtc tRHdVje: NewTransaction (8)
SEND WebRtc tV1dqvu: NewTransaction (8)
SEND WebRtc w0XFOiR: NewTransaction (8)
SEND WebRtc rOgUpNf: NewTransaction (8)
SEND WebRtc V1oqhXg: NewTransaction (8)
SEND WebRtc R95PNbU: NewTransaction (8)
SEND WebRtc 7BOpPhA: NewTransaction (8)
SEND WebRtc bs0mHfT: NewTransaction (8)
Invalid transaction '1b178b0d82055e23f006cd02f1c8273fe496e5bdab2dcf97aeabb13e4ccf3031': expired sequence 114748
Invalid transaction '8be20ccbdf28732ec54355fcbcf9e594219d2d6b069cecf009ac201272a033b1': expired sequence 114676
Invalid transaction '035fb4f83cbda146eef215ee09d3dfe6bbbd0caa9103ba207866869210d2b309': expired sequence 114835
Invalid transaction '85d55a51954b73219b079241c02382f01052a002abaa6415ab8a51bc31ec909b': expired sequence 114924
Invalid transaction '70d57c16bf369b344751b811732d813098cfc8d25b6a75fac380c86d2b31ee01': expired sequence 114925
/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:316
                            throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map');
                                  ^

Error: nullifierToNote mappings must have a corresponding noteToNullifier map
    at /usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:316:35
    at async LevelupDatabase.withTransaction (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/storage/database/database.js:39:28)
```
And from third:

```
Accepted tx 63bd16402bdedce5db4d76effa119d15dc7a03ace7bfcc9fc127c8b25704c491, poolsize 2385
SEND WebRtc kkkEWU2: NewTransaction (8) (repeated 2 times)
RECV WebRtc kkkEWU2: NewTransaction (8)
RECV WebRtc kkkEWU2: NewTransaction (8)
RECV WebRtc kkkEWU2: NewTransaction (8)
SEND WebRtc PbvrcmG: NewTransaction (8) (repeated 8 times)
RECV WebRtc PbvrcmG: NewTransaction (8)
SEND WebRtc hxoiVjH: NewTransaction (8) (repeated 8 times)
RECV WebRtc hxoiVjH: NewTransaction (8)
SEND WebRtc mGMpkDN: NewTransaction (8) (repeated 4 times)
RECV WebRtc mGMpkDN: NewTransaction (8)
SEND WebRtc u06duGS: NewTransaction (8) (repeated 8 times)
RECV WebRtc u06duGS: NewTransaction (8)
SEND WebRtc s/XWFWz: NewTransaction (8) (repeated 12 times)
RECV WebRtc s/XWFWz: NewTransaction (8)
RECV WebRtc s/XWFWz: NewTransaction (8)
Invalid transaction 'a2847f0f318fcae8d1e0b66561376296a967adcb28ab481a75ef7a71b76660fb': expired sequence 114825
Invalid transaction '2ee070278d6e8375543cdfba941fafcc7bbab920375158dc74c568649e2ed6ea': expired sequence 114785
Invalid transaction '8d87f11ba5645337295171aa5432bd749d5181537434458d8e50cb3f3adaba87': expired sequence 114785
RECV WebRtc kkkEWU2: NewTransaction (8)
RECV WebRtc PbvrcmG: PeerListRequest (10)
RECV WebRtc PbvrcmG: NewTransaction (8)
RECV WebRtc PbvrcmG: NewTransaction (8)
RECV WebRtc PbvrcmG: NewTransaction (8)
RECV WebRtc PbvrcmG: NewTransaction (8)
RECV WebRtc PbvrcmG: NewTransaction (8)
SEND WebRtc GpIsNxH: NewTransaction (8) (repeated 9 times)
RECV WebRtc GpIsNxH: NewTransaction (8)
RECV WebRtc GpIsNxH: NewTransaction (8)
RECV WebRtc hxoiVjH: NewTransaction (8)
RECV WebRtc hxoiVjH: NewTransaction (8)
CONN WebSocket unidentified STATE CONNECTING -> WAITING_FOR_IDENTITY
SEND WebSocket unidentified: Identify (6)
Received non-buffer message {"type":0,"destinationIdentity":null,"disconnectUntil":1659085768449,"reason":1,"sourceIdentity":"qk+TVjAnCzXwqfaiooBou2Cs4ZnutIbFEwMkghx2A2A="}
CONN WebSocket unidentified STATE WAITING_FOR_IDENTITY -> DISCONNECTED
Connection closing WebSocket for qk+TVjA (crptnwb): Received non-buffer message
PEER qk+TVjA (crptnwb) STATE CONNECTING -> DISCONNECTED
Invalid transaction '4419b6134d4c00043b138df93e12e231abfa991e716598f92a0ab4509f985743': expired sequence 114785
Invalid transaction 'ad55422c31a543e525e67f71ea74dd7d5f44966774188dd4c9b5020e26eba5f6': expired sequence 114785
Invalid transaction 'c344763ace747e2ba6a0c2cfe5318f3778fedcc8d511d7b810f2548f834f0f10': expired sequence 114785
Invalid transaction '2eba75043d0cc934df76ba64c672576abcb0870895cb775511f41a56fb0f38b1': expired sequence 114785
Invalid transaction 'abd57441555652e095f19a66092d5732d75002a93da4bcff733ea73b82884e37': expired sequence 114785
Invalid transaction 'a0af01828b1a09a512304469df79e40bfdccf410a05673a608420e0905169583': expired sequence 114785
RECV WebRtc GpIsNxH: NewTransaction (8)
SEND WebRtc 89bwcJS: NewTransaction (8)
RECV WebRtc 89bwcJS: NewTransaction (8)
Invalid transaction 'e1e1a264ef466c571f98775bfbc5f97cd21c72b9cf2af8a74caa0aee397bd9d3': expired sequence 114607
Invalid transaction '597609b3b677fc74b902db95614b18b322e7bc9bf7614f06b71d4fb8e0c10cd6': expired sequence 114598
Invalid transaction '9c6733a73bcc54f402a51ddfbe1783804b100a35fbaee1836d10889c8c0f753d': expired sequence 114785
RECV WebRtc kkkEWU2: NewTransaction (8)
Invalid transaction 'f2c32489d20a1f67e61f0872b514eac7748ae076f8832f2f4c971bc756741022': expired sequence 114602
Invalid transaction 'c3e9d8802c0f1dbefd136859e2ed646e48599a97b3e99674ac5d7057e67c9e50': expired sequence 114785
RECV WebRtc GpIsNxH: NewTransaction (8)
RECV WebRtc GpIsNxH: NewTransaction (8)
Invalid transaction '180ee710039b32a0b4172bb81412ddf089dea64606dd4d88bb1654fd07bedbc7': expired sequence 114785
Invalid transaction '1eda2f7c2d118b8f0f872112fa5016e77341c459d845d9b4f5a2702f710df667': expired sequence 114785
Invalid transaction 'f209f4bbafd654c98743debbac4cde8165c3137c558385d7a4052dce1c53c116': expired sequence 114827
Invalid transaction '7978fc502fabeede458f476fdafa0537568299bc683ecadb8a09f64031e45b5d': expired sequence 114785
RECV WebRtc 89bwcJS: NewTransaction (8)
Invalid transaction 'cc9b08e889baed96c6913825608200259ff67ca182578eac7863f5daea743dfe': expired sequence 114785
RECV WebRtc 89bwcJS: NewTransaction (8)
RECV WebRtc 89bwcJS: NewTransaction (8)
Invalid transaction 'f79eb1e5337a03c0635978e08ed61931585cec35a51e9e2860f267d5718b9858': expired sequence 114785
Invalid transaction '29ebb48375ff537d1496499b6db96645d17502ea6e4fc91b8198c560373756fd': expired sequence 114785
Invalid transaction 'd96090d1179a41990bb4ff4cdc8659919ce19c63ba6e3fb3935e3b4247707127': expired sequence 114785
Invalid transaction '49c677a40514aa42b20cb30a4d5bad10a3128bd1269e121c5082a21fe3c9e3d9': expired sequence 114785
Invalid transaction '25bcce71ee9e92049848f562ebc310d315b37cdc22874fcca1f78170286605ab': expired sequence 114785
Invalid transaction '67ce919bfdc4f89984acaac160a962b5eaa5a116439c23ad5c74611f11d32778': expired sequence 114785
RECV WebRtc s/XWFWz: NewTransaction (8)
RECV WebRtc s/XWFWz: NewTransaction (8)
RECV WebRtc 89bwcJS: NewTransaction (8)
RECV WebRtc s/XWFWz: NewTransaction (8)
Invalid transaction '8862c7db77704e5a85b4e66a7c0965e55bae48eb5f34f87244ad24c6addeaf4b': expired sequence 114785
Invalid transaction '6dbf2e0a4ce801d5c04f050b6b1ab757f32f5a79c24a383c479d5007df481b9a': expired sequence 114785
Invalid transaction '7e346c07ccddc252b5c35af4317df3c7821b772a1da7a40883e984b0af929ff1': expired sequence 114785
Invalid transaction 'f570c59562bf72eaff70a9014ce8a5008be87b3b1778e0f8cf8d1b175a2fcee2': expired sequence 114739
Invalid transaction 'c522e90532b643957a953337a152023a9c4318a0472ed572038bd17101e3a549': expired sequence 114785
Invalid transaction '0a9fbb6e5a4260cc06e84a166b07926d27cf7c192aef7d955c67e3d106a0545e': expired sequence 114785
SEND WebRtc dPImMCL: NewTransaction (8) (repeated 13 times)
RECV WebRtc dPImMCL: NewTransaction (8)
SEND WebRtc Ut9YZyI: NewTransaction (8) (repeated 8 times)
RECV WebRtc Ut9YZyI: NewTransaction (8)
Invalid transaction '50883d4225d141b03d73a2c009075aba1cd716eeff11394bed17d9b2f3246643': expired sequence 114746
Invalid transaction '2a1c6c1eecd3c35d105e18289ec0c5abc8732008d84700292eadbfe84deec778': expired sequence 114785
RECV WebRtc Ut9YZyI: NewTransaction (8)
Invalid transaction '9dac1fffd9b7caeec20458c0af188cca985e848472e18b66ebfa374be074adcc': expired sequence 114785
Invalid transaction '4ad825b499b3cc658abfa8bdfb154841eae9cf3fc114e8059cf0ca31ecb96490': expired sequence 114848
RECV WebRtc dPImMCL: NewTransaction (8)
Invalid transaction '5adb55c4a1753e8bd9a7d5a36c5f2d0f078e3767945125c9085771089b532995': expired sequence 114776
Invalid transaction '8e5d36527e798ee2c07c6d6b782c2c03777f78f2944e64f9a50fde3108ac3dbc': expired sequence 114862
Invalid transaction 'e30409dc49fc7a55818ef1452af128403f581130240eeb3a080daca9bb8860af': expired sequence 114785
/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:316
                            throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map');
                                  ^

Error: nullifierToNote mappings must have a corresponding noteToNullifier map
    at /usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:316:35
    at async LevelupDatabase.withTransaction (/usr/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/storage/database/database.js:39:28)
```

**Comment by lwisne** [comment]
Sorry for the pain here - we are still trying to track this down.

Thanks for the detailed logs!

**Comment by hughy** [comment]
Errors like this one should be resolved by the Wallet 2.0 refactor released in [v0.1.48](https://github.com/iron-fish/ironfish/releases/tag/v0.1.48)

Please open a new issue if you run into this again!
